.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
```

Comment:
There's also a docker ecosystem here.

Consider adding ...
```
  - package-ecosystem: "docker"
    # Look for a `Dockerfile` in the `root` directory
    directory: "/"
    schedule:
      interval: "weekly"
```

We might need to tweak this `directory` setting later.